credentials/apps/badges/models.py

```diff
@@ -683,7 +684,7 @@ class AccredibleAPIConfig(TimeStampedModel):
     Accredible API configuration.
     """
 
-    name = models.CharField(max_length=255, help_text=_("Accredible API configuration name."), null=True, blank=True)
+    name = models.CharField(max_length=255, help_text=_("Accredible API configuration name."), null=True, blank=False)
```

Comment:
Do we need new migration on this change?

Reply:
I can see result without creating migration, let me check

Reply:
I'll push new migration